src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_TypeChar.cs

```diff
@@ -313,7 +313,7 @@ private bool IsTextualTriggerCharacter(ICompletionService completionService, cha
             return completionService.IsTriggerCharacter(previousPosition.Snapshot.AsText(), previousPosition, GetCompletionProviders(), options);
         }
 
-        private bool IsCommitCharacter(char ch)
+        private bool IsCommitCharacter(char typedChar)
```

Comment:
Currently Controller_CaretPositionChanged.cs does not wait for model, do you mind me placing
```
// TODO(cyrusn): Find a way to allow the user to cancel out of this.
```
as you did in Controller_TypedChar.cs?